website/pages/docs/plugins/source-plugin-release-stages.md

```diff
@@ -1,27 +1,18 @@
 # Source Plugin Release Stages
 
-[Official](source) source plugins go through multiple release stages: Preview, and GA.
+[Official](source) source plugins go through two release stages: Preview, and GA.
 
-## Comparison Matrix
+Both Preview and GA plugins follow [semantic versioning](#semantic-versioning).
+The main differences between the two stages are:
 
-See the following comparison matrix to understand differences and expectations.
-
-|                                | Preview | GA                  |
-| ------------------------------ | ------- | ------------------- |
-| Spec (Config) Breaking Changes | Yes     | Semantic Versioning |
-| Schema Changes                 | Yes     | Semantic Versioning |
-| Long Term Support (Bug fixes)   | No      | Yes                 |
-
-- Spec (Config) Breaking Changes - While a plugin is in Alpha, we might change how the plugin spec is defined for a plugin in a backward-incompatible way. Starting Beta we guarantee no **breaking** changes (we will still be adding features).
-- Schema Changes - While a plugin is in Alpha, breaking changes to schema are possible without the plugin following semantic versioning. Starting Beta, we do our best to not break the schema in non-backward compatible way, but if we do then plugin will follow [semantic versioning](#semantic-versioning).
-- Long Term Support - When a plugin is an Alpha, it is still experimental, and we might decide not to support it and/or deprecate it (including bug fixes). When a plugin is entering Beta phase, we guarantee long-term support with community Discord and bug fixes (additional features and resources roadmap can be found in GitHub issues).
+1. Preview plugins are still experimental and may have frequent breaking changes.
+2. Preview plugins might get deprecated due to lack of usage.
+3. Long Term Support with community Discord and bug fixes is only guaranteed for GA plugins.
 
 ## Semantic Versioning
 
-Official plugins follow semantic versioning with the following logic:
-
-For version `x.y.z`:
+For version `Major.Minor.Patch`:
 
-- `x`: `Spec` configuration changed in backward incompatible way.
-- `y`: Incompatible schema where one or more columns/tables changed in a way that will require you to re-create/delete columns/tables. Documentation on which columns/tables are affected will be documented in the plugin release notes.
-- `z`: No breaking changes. Features and/or bug fixes are documented in the plugin release notes.
+- `Major` is incremented when there are breaking changes.
+- `Minor` is incremented when we add features in a backwards compatible way.
```

Comment:
No examples is intentional in order not to commit to what constitutes as a breaking change